CONTRIBUTING.md

```diff
@@ -39,9 +39,34 @@ All code submissions should be submitted with regression test cases, and will be
 
 - We expect contributors to be actively involved in quality assurance.
 - We expect contributors to be actively involved in quality assurance.
+- We expect contributors to be actively involved in quality assurance.
```

Comment:
like this seem a typo, maybe :
- **We expect contributors to be actively involved in quality assurance.**
- :exclamation:  We expect contributors to be actively involved in quality assurance.  
- :star: We expect contributors to be actively involved in quality assurance. :star: